initial/src/main/java/com/example/springboot/HelloController.java

```diff
@@ -1,14 +1,40 @@
 package com.example.springboot;
 
+import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.bind.annotation.RequestMapping;
+import java.util.Random;
 
 @RestController
 public class HelloController {
 
 	@RequestMapping("/")
-	public String index() {
-		return "Hello Ethan Wei, welcome to Spring Boot!";
+	//create algorithm under controller to send the calculated value to the server
+	public String index(String[] args) {
+		Random rand = new Random();
+		int upperbound = 25;
+		int num1 = rand.nextInt(upperbound);
+		int num2 = rand.nextInt(upperbound);
+		String[] randOpr = {"+","-","*","/"};
```

Comment:
1. I would suggest making `randOpr` a class-level variable and make it `private static final` as I see this array is a read-only array. 
2. `randOpr` can be updated to `operators` maybe? as it doesn't look like random to me :P

Reply:
1. got it! 
2. makes sense! I though to name it that because it is chosen randomly, but that's a good call out that the elements themselves are not actually random.